README.md

```diff
@@ -69,6 +69,28 @@ To enable logging on the client-side, just:
 
     (setq lsp-log-io t)
 
+### IntelliJ Setup
+
+First of all, ensure you have the [LSP
+Support](https://github.com/gtache/intellij-lsp) plugin installed. If
+you don't, you can simply navigate to:
+
+    Preferences > Plugins > Browse Repositories
+
+Search for "LSP Support" and install the respective plugin.
+
+Restart IntelliJ, then navigate to:
+
+    Preferences > Languages and Frameworks > Language Server Protocol > Server Definitions
+
+There you can instruct IntelliJ on how to start the server. Select
+`Raw Command`, set `erl` as the extension, then add as the command:
+
+    /ABSOLUTE/PATH/TO/erlang_ls/_build/debug/bin/erlang_ls --stdio
```

Comment:
This has to be updated now. Anyway, I would like to wait with this until we get IntelliJ working.